src/runtime/manifest.ts

```diff
@@ -19,6 +19,16 @@
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
+/**
+ * One or more event filters restrict the set of events delivered to an EventTrigger.
+ */
+
+export interface EventFilter {
```

Comment:
I think this goes away in a merge, right? We're going to use the same format as the channel API where we have eventFilters and eventPathPatternFilters

Reply:
Hm... I think it's a bad merge. Removed.